content/SCALE/SCALETutorials/Credentials/DirectoryServices/ConfigADSCALE.md

```diff
@@ -26,50 +26,90 @@ Users can configure AD services on Windows or Unix-like operating systems using
 
 To configure an AD connection, you must know the AD controller domain and the AD system account credentials.
 
-### Preparation
+### Preparing to Configure AD in SCALE
 
-Users can take a few steps before configuring Active Directory to ensure the connection process goes smoothly.
+Users can take a few steps before configuring Active Directory (AD) to ensure the connection process goes smoothly.
+
+1. Locate the Relative Distinguished Name (RDN) of the site object. 
+   This is the first component of the distinguishedName property configured in AD.
+   {{< expand "Need AD Help?" "v" >}}
+   To locate the distinguished name: 
+
+   1. Log into AD.
+   2. Click on **View** and select **Advanced Features**.
+   3. Navigate to and right-click on the organizational unit (OU) where you want to read users, service or computer accounts, then select **Properties**.
+   4. Select the **Attribute Editor** tab on the **OU Properties** window.
+   5. Click on **distinguishedName** to highlight it then click **View**.
+   {{< /expand >}}
+
+2. Obtain the AD admin account name and password.  
+
+3. [Verify name resolution](#verifying-name-resolution)
+
+5. [Set type synchronization](#setting-time-synchronization)
+
+After taking these actions, you can [connect to the Active Directory domain](#connecting-to-the-active-directory-domain).
 
 ### Verifying Name Resolution
 
-To confirm that name resolution is functioning, go to **System Settings > Shell** and use `ping` to check the connection to the AD domain controller.
+To confirm that name resolution is functioning, you can use the **Shell** and issue a `ping` command and a command to check network SRV records and verify DNS resolution.
 
-![ShellDomainPingSCALE](/images/SCALE/ShellDomainPingSCALE.png "Pinging a Domain Controller")
+To use `ping` to verify name resolution:
 
-When TrueNAS sends and receives packets without loss, the connection is verified. Press <kbd>Ctrl + C</kbd> to cancel the `ping`.
+1. Go to **System Settings > Shell** and type `ping` to check the connection to the AD domain controller. 
+   The domain controller manages or restricts access to domain resources by authenticating user identity from one domain to the other through a login credentials, and it prevents unauthorized access to these resources. The domain controller applies security policies to request-for-access domain resources.
```

Comment:
Ping isn't a good tool for this. `dig` is better. You should be able to get SRV records for the AD domain prior to joining. Some users misconfigure DNS in subtle ways (using non-AD nameservers)

Reply:
I was not able to get SRV record back when I set up AD. Is that common?

Reply:
What do you mean by get SRV record back? Perhaps DNS was misconfigured in your case (for example having non-AD nameservers). You should always be able to retrieve SRV records.

Reply:
I didn't see the one-to-one in the old procedure, so my guess is it goes in the **Domain** field.